app/results.py

```diff
@@ -261,5 +264,6 @@ def get(self):
                                query_location_original=
```

Comment:
Looks like these query_location_original and query_location parameters, and also query_location *variable*, are no longer used. Can you delete them?